README.md

```diff
@@ -2,22 +2,25 @@
 
 ## Requirements
 
-1. Postgres
+1. Yarn (https://classic.yarnpkg.com/en/docs/install/)
+1. Postgres (https://www.postgresql.org/download/)
    - when installing, use the default password "postgres"
    - for macOS users, suggested installation is through homebrew
 2. TypeORM Global Installation (`yarn add typeorm -g`)
    - this will install the TypeORM CLI as well
-3. Redis
+3. Redis (https://redis.io/download)
```

Comment:
Thanks for adding this 😅  Do you mind changing to "Local Redis Instance"?

Reply:
How come "Local Redis Instance"?

Reply:
It's just to be clear that it's being used locally- since the deployment one will be the Docker version

Reply:
Hmm I'm not sure I understand, usually for local development it's suggested that the services you use are local instances. Deployment is a whole other story right

Reply:
Yeah! Like the setup you follow here is so that you can get a local Redis instance running instead of using the Redis with Docker for development

Reply:
Oh, wait rather than `deployment`, you meant development with Docker? Yah docker development is covered in other section of the ReadMe

https://github.com/ubclaunchpad/PromoPal-backend/blob/7e3a4b421efc01e8161f37df7e5d3558cf9db9a4/README.md#local-development-with-docker

Otherwise, I think we would need to say "Local Postgres Instance" then

Reply:
Ahh when I said deployment, I meant that the Redis we'll use when our app deployed, is the Redis associated with Docker! And in that case, I think we should follow "Local Postgres Instance" as well- so future devs don't get mixed up